### PR TITLE
Cloudformation: use is_boto3_error_message

### PR DESCRIPTION
**Pull request by tremble**
##### SUMMARY

use our is_boto3_error_message helper to handle 'expected' failures

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

cloudformation

##### ADDITIONAL INFORMATION

### CODE REVIEW

File: plugins/modules/cloudformation.py [hunk 1]
```diff
@@ -466,12 +466,10 @@ def create_changeset(module, stack_params, cfn, events_limit):
             result['warnings'] = ['Created changeset named %s for stack %s' % (changeset_name, stack_params['StackName']),
                                   'You can execute it using: aws cloudformation execute-change-set --change-set-name %s' % cs['Id'],
                                   'NOTE that dependencies on this stack might fail due to pending changes!']
+    except is_boto3_error_message('No updates are to be performed.':
```
Comment (jillr) [inline plugins/modules/cloudformation.py:469]:
```suggestion
    except is_boto3_error_message('No updates are to be performed.'):
```

File: plugins/modules/cloudformation.py [hunk 2]
```diff
@@ -491,12 +489,10 @@ def update_stack(module, stack_params, cfn, events_limit):
     try:
         cfn.update_stack(**stack_params)
         result = stack_operation(cfn, stack_params['StackName'], 'UPDATE', events_limit, stack_params.get('ClientRequestToken', None))
+    except is_boto3_error_message('No updates are to be performed.':
```
Comment (jillr) [inline plugins/modules/cloudformation.py:492]:
```suggestion
    except is_boto3_error_message('No updates are to be performed.'):
```

**Review by jillr — APPROVED** [review]
lgtm when the syntax errors are fixed

**Comment by tremble** [comment]
recheck

**Pull request merged**